resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -179,22 +131,85 @@ public DnsNameResolver(
      * @param nameServerAddresses the addresses of the DNS server. For each DNS query, a new stream is created from
      *                            this to determine which DNS server should be contacted for the next retry in case
      *                            of failure.
+     * @param minTtl
+     * @param maxTtl
+     * @param negativeTtl
+     * @param queryTimeoutMillis
+     * @param resolvedAddressTypes
+     * @param recursionDesired
+     * @param maxQueriesPerResolve
+     * @param traceEnabled
+     * @param maxPayloadSize
+     * @param optResourceEnabled
+     * @param hostsFileEntriesResolver
      */
     public DnsNameResolver(
```

Comment:
Could we decide once I/we have a solution for #4568?
I mean I'd rather not have to fork because I can't extend or plug hooks.